README.md

```diff
@@ -17,7 +17,7 @@ For information about HDF5 filter plugins, see...
 
 - https://support.hdfgroup.org/HDF5/doc/Advanced/DynamicallyLoadedFilters
 
-This H5Z-ZFP plugin supports ZFP versions 0.5.0 through 0.5.5.
+This H5Z-ZFP plugin supports ZFP versions 0.5.0 through 1.0.0.
```

Comment:
Maybe just say "the latest release" instead of 1.0.0?

Reply:
```suggestion
This H5Z-ZFP plugin supports ZFP versions 0.5.0 and newer.
```